README.md

```diff
@@ -1,5 +1,294 @@
-This is an app where members can explore & find other members in your community. Perfect for events & programs!
+# The Orbit Member Directory
 
-To run locally:
+1. [What is it?](#what-is-it)
+2. [Technical Overview](#technical-overview)
+   1. [Data](#data)
+   2. [Authentication](#authentication)
+      1. [Securing Frontend Pages](#securing-frontend-pages)
+      2. [Securing API Endpoints](#securing-api-endpoints)
+   3. [API Protection](#api-protection)
+3. [Local Setup](#local-setup)
+   1. [Environment Variables](#environment-variables)
+      1. [For Orbit](#for-orbit)
+      2. [For Emails](#for-emails)
+      3. [For the Database](#for-the-database)
+      4. [For next-auth](#for-next-auth)
+   2. [Initial data](#initial-data)
+   3. [Running the server, logging in](#running-the-server-logging-in)
+4. [Setting up your own instance](#setting-up-your-own-instance)
+   1. [Branding](#branding)
+      1. [Update brand colours](#update-brand-colours)
+      2. [Update brand images](#update-brand-images)
+      3. [Update SEO](#update-seo)
+   2. [Hosting (Recommended)](#hosting-recommended)
+5. [General tips](#general-tips)
+   1. [Changing the DB structure](#changing-the-db-structure)
 
-`yarn dev`
+## What is it?
+
+This app lets you host a subset of the members in your community and let them share their social profiles & a short bio about themselves. It is ideal for promoting engagement and fostering relationships between members.
+
+![The landing page of the member directory, with Orbit branding.](https://github.com/orbit-love/member-directory/assets/45462299/a1bad632-4437-4aaf-9459-e310e17c04c2)
+
+## Technical Overview
+
+Diagram here...
+
+### Data
+
+[Prisma](https://www.prisma.io/) is used to interface with a Postgres database that exists between the Orbit API and the member directory. This provides control over the data, allowing for manipulation independent of your main data store n Orbit. For example, we've added "bio" and "admin" fields not required by the Orbit application.
+
+Data is fetched from Orbit using the /api/populate-members-table API route. This process fetches members fitting specific criteria (presence of an email and a defined tag in environment variables) and upserts them into Prisma. Members that no longer meet these criteria and still exist in Prisma are removed.
+
```

Comment:
Some suggestions
1. Use simpler words (fitting, upserts)
2. Reduce complexity (This process fetches members → we fetch members)